core/server/data/meta/context_object.js

```diff
@@ -1,12 +1,12 @@
-var config = require('../../config');
+var config = require('../../config'),
+    _      = require('lodash');
 
 function getContextObject(data, context) {
     var blog = config.theme,
         contextObject;
 
-    context = context === 'page' ? 'post' : context;
+    context = _.includes(context, 'page') || _.includes(context, 'amp') ? 'post' : context;
```

Comment:
I know. But I think in this case it doesn't do any harm. The tests on master would fail without `context_object.js`.... 😔